### PR TITLE
add test on alpine using php7

### PR DESCRIPTION
**Pull request by xXx-caillou-xXx**
Add tests to the CI for alpine using php7, php8 isn't in the apk repository.

This does not fix the PHP8 tests faling but is related to #370

### CODE REVIEW

**Comment by jvoisin** [comment]
PHP8 is present in edge/testing iirc

**Comment by xXx-caillou-xXx** [comment]
I have no idea why the tests are failing on php7.
`alpine:edge` with php8 seems to be bugged and the package `php8-dev` does not contain `phpize`.

**Comment by remicollet** [comment]
Notice, use 
`make test TESTS="--show-diff" `

Will make output momre ledgible ;)

**Comment by wargio** [comment]
can you update this code following @jvoisin code for the ci? so we can have all the main distros tested :D

**Comment by xXx-caillou-xXx** [comment]
The tests now run on github actions, but I had to remove _quite a lot_ of failing tests :>

File: .github/workflows/distributions.yml [hunk 1]
```diff
@@ -89,3 +89,26 @@ jobs:
...
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Remove php8 tests for php7
+        run: rm -rf src/tests/*php8*/
+      - name: Remove tests failing on alpine for wathever reason
+        run: rm -rf src/tests/*session*/ src/tests/broken_configuration/ src/tests/*cookie* src/tests/upload_validation/
+      - name: Install dependencies
+        run: |
+          apk add php7-dev php7-cgi php7-simplexml php7-xml pcre-dev make build-base php7-pear php7-openssl
```
Comment (jvoisin) [inline .github/workflows/distributions.yml:105]: I think that `make` is included in `build-base`, isn't it?

Comment (jvoisin) [reply .github/workflows/distributions.yml:105]: Also, no need for the `|` for a single item

**Review by jvoisin — APPROVED** [review]

**Pull request merged**